frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.tsx

```diff
@@ -0,0 +1,233 @@
+      const leader = {
+        shadow: {
+          enabled: true,
+          color: styles.leaderNodeColor,
+          size: 15,
+          x: 1,
+          y: 1
+        }
+      };
+      const oldest = {
+        borderWidth: 3,
+        borderWidthSelected: 0,
+        color: {
+          border: styles.oldestNodeColor,
+          highlight: { border: styles.oldestNodeColor },
+          hover: { border: styles.oldestNodeColor }
+        },
+        shapeProperties: { borderDashes: [10, 10] }
+      };
+      //eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const graph: any = {
+        nodes: [
+          {
+            id: 0,
+            label: "Akka Cluster",
+            image: clusterUrl,
+            size: 40,
+            borderWidth: 1,
+            borderWidthSelected: 2,
+            color: {
+              border: styles.primaryColor,
+              background: styles.secondaryColor,
+              highlight: {
+                border: styles.primaryColor,
+                background: styles.secondaryColorLighter
+              },
+              hover: {
+                border: styles.primaryColor,
+                background: styles.secondaryColorLighter
+              }
+            }
+          }
+        ],
+        edges: []
+      };
+
+      if (!clusterData || !clusterData.members) {
+        return graph;
+      }
+
+      clusterData.members.forEach((member, index) => {
+        const memberTitle = (
+          <React.Fragment>
+            <div>
+              <span className="font-bold disp-inline-blk">node : </span>
+              <span
+                className={
+                  "status disp-inline-blk " + member.status.toLowerCase()
+                }
+              ></span>
+              <span className="disp-inline-blk">{member.node}</span>
+            </div>
+            <div>
+              <span className="font-bold">nodeUid : </span>
+              <span>{member.nodeUid}</span>
+            </div>
+            <div>
+              <span className="font-bold">status : </span>
+              <span>{member.status.toLowerCase()}</span>
+            </div>
+            <div>
+              <span className="font-bold">roles : </span>
+              <span>{member.roles.join(", ")}</span>
+            </div>
+            <div>
+              {member.node === clusterData.oldest && (
+                <span className="type oldest disp-inline-blk">Oldest</span>
+              )}
+              {member.node === clusterData.leader && (
+                <span className="type leader disp-inline-blk">Leader</span>
+              )}
+            </div>
+          </React.Fragment>
+        );
+
+        const memberConfig = {
+          id: index + 1,
+          label: "<b>o </b>" + member.node.split("://")[1],
+          image: nodeUrl,
+          title: ReactDOMServer.renderToString(memberTitle),
+          font: {
+            bold: {
+              color: styles[`status${member.status}Color`],
```

Comment:
I assume some CSS here could get the desired circle effect as well instead of a ::before, just leave the `<b />` tag empty, and have the CSS here be something like:

```bold: {
  width: '1rem',
  height: '1rem',
  verticalAlign: 'middle',
  borderRadius: '.5rem',
  backgroundColor: styles[`status${member.status}Color`],
  marginRight: '.5rem',
  display: 'inline-block'
}```

Reply:
As I mentioned in the previous comment, vis graphs don't seem to support it yet. Might have to look out for work-arounds.